src/utils.sh

```diff
@@ -23,4 +25,32 @@ function kworkflow-help()
     "\t                             the mailing list. \"-a\" also\n" \
     "\t                             prints files authors\n" \
     "\thelp,h - displays this help mesage"
+    "\tcodestyle - Apply checkpatch on directory or file\n" \
```

Comment:
you introduced a tiny bug here, if you run the tests you will see:

./src/utils.sh: line 29: \tcodestyle - Apply checkpatch on directory or file\n: command not found

Additionally, why you moved the other help descriptions? You just had to add you new entry here. Finally, I think you forgot to rebase with the unstable.